api/qanpb/filters.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package qan;
+
+option go_package = "qanpb";
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+// Filters serves list of names of metrics.
+service Filters {
+  // GetFilters gets map of metrics names.
+  rpc GetFilters(FiltersRequest) returns (FiltersReply) {
+    option (google.api.http) = {
+      post: "/v1/qan/GetFilters"
+      body: "*"
+    };
+  }
+}
+
+// FiltersRequest contains period for which we need filters.
+message FiltersRequest {
+  google.protobuf.Timestamp from = 1;
```

Comment:
Can those values be empty? If not, can we add validators?

Reply:
A good point. Added validation in qan-api2 https://github.com/percona/qan-api2/blob/ce300d1f12e4c7f3ea07c26699ac34a9d205baf1/services/analytics/filters.go#L30